namespaces/kubernetes.yml

```diff
@@ -11,34 +11,39 @@ namespace:
       enabled: true
     description: >
       The name of the pod
+      In case this logs carries a kubernetes event that was triggered by a pod only, it is overwritten by that pod's pod_name.
```

Comment:
Could lines 13 & 14 be condensed to just: `The name of the pod associated with the event.`